airgun/widgets.py

```diff
@@ -1321,6 +1321,42 @@ class EditableEntryMultiCheckbox(EditableEntry):
     edit_field = CheckboxGroup(locator='.//form')
 
 
+class EditableEntryMultiTextInput(GenericLocatorWidget):
```

Comment:
This is the same thing as in #325 , right? If so, I think it should be removed from this PR and this PR should be merged only after #325 is merged.

Reply:
I made that commit part of this PR - due to some issues with branching, it had to be done. I think the best way to handle this would be to close https://github.com/SatelliteQE/airgun/pull/325/commits and accept this PR as-is. Agree?

Reply:
Hmm, if you just copy/pasted content of other PR here, so yeah, we can close #325 and address comments in it there, but again, you adding mess where it should not be added in the first place.

Reply:
Actually I rebased this PR to contain the other PR. Let's close that one, I will resolve comments from @ldjebran here.